powertools-idempotency/powertools-idempotency-core/src/main/java/software/amazon/lambda/powertools/idempotency/persistence/DataRecord.java

```diff
@@ -131,13 +131,22 @@ public int hashCode() {
         return Objects.hash(idempotencyKey, status, expiryTimestamp, responseData, payloadHash);
     }
 
+    @Override
+    public String toString() {
+        return "DataRecord{" +
+                "idempotencyKey='" + idempotencyKey + '\'' +
+                ", status='" + status + '\'' +
+                ", expiryTimestamp=" + expiryTimestamp +
+                ", payloadHash='" + payloadHash + '\'' +
+                '}';
+    }
 
     /**
      * Status of the record:
      * <ul>
-     *  <li>INPROGRESS: record initialized when function starts</li>
-     *  <li>COMPLETED: record updated with the result of the function when it ends</li>
-     *  <li>EXPIRED: record expired, idempotency will not happen</li>
+     * <li>INPROGRESS: record initialized when function starts</li>
```

Comment:
Extremely minor, but shouldn't this stay indented since it's a html tag and the list item is a child of the unordered list?

Alternatively, can this be converted to markdown notation instead?

Reply:
This was done because the VSCode formatter is slightly different from the IntelliJ Idea formatter that was used by the previous author on this file. This is related to the fact that we don't have CLI-based formatting for the project that works independent of a specific IDE. 

There is an issue for this already: https://github.com/aws-powertools/powertools-lambda-java/issues/1345. Once we decide for a consistent formatting solution these diffs should not appear anymore.